CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+# Contributing to the dbt Artifacts Package
+
+Thank you for your interest in contributing to the dbt Artifacts Package! We welcome contributions of all kinds,
+including bug reports, feature requests, and pull requests. Please read this document to learn how to contribute.
+
+## Reporting Bugs :bug:
+
+If you find a bug, please [open an issue](https://github.com/brooklyn-data/dbt_artifacts/issues/new) and describe the
```

Comment:
It would be great to have some PR and issue templates at some stage (probably already on the roadmap, but flagging in case not).

Reply:
Great idea, created Issues for both!